src/utils/Profiler.ts

```diff
@@ -1,8 +1,14 @@
 import { performance } from "node:perf_hooks";
-import { Logger } from "winston";
 import crypto from "crypto";
 import { DefaultLogLevels } from "./LogUtils";
 
+type Logger = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  debug: (...args: any[]) => void;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  warn: (...args: any[]) => void;
```

Comment:
Could we use `unknown` here instead of `any` ?

Reply:
i want to keep it as close to most implementations which is usually types as `any` (winston ans global `console` in node and browser contexts type it this way).
Also using unknown is actually a bit too narrow, stopping us from using the spread operator